grass7/imagery/i.sentinel/i.sentinel.parallel.download/i.sentinel.parallel.download.html

```diff
@@ -1,7 +1,9 @@
 <h2>DESCRIPTION</h2>
 
 <em>i.sentinel.parallel.download</em> downloads Sentinel-2 data defined
-by scene names in parallel using <em>i.sentinel.download</em>.
+by scene names in parallel using <em>i.sentinel.download</em>.<br>
+If no scene names are passed <em>i.sentinel.parallel.download</em>
+downloads data for the computational region set in the mapset.
```

Comment:
```suggestion
If no scene names are specified <em>i.sentinel.parallel.download</em>
downloads data for the computational region set in the mapset; i.e, only
products which footprint intersects the current computation region extent
(area of interest, AOI) are selected.
```

(addition based on https://grass.osgeo.org/grass7/manuals/addons/i.sentinel.download.html#notes)